package.json

```diff
@@ -105,6 +105,7 @@
     "@loadable/component": "^5.12.0",
     "@loadable/server": "^5.12.0",
     "@loadable/webpack-plugin": "^5.12.0",
+    "aws-embedded-metrics": "^2.0.0",
```

Comment:
Does this get included in the bundle and shipped to the client?

Reply:
It's only imported in the server so I would have thought not, however I will confirm